src/lib/lifeCycle.ts

```diff
@@ -85,6 +122,12 @@ const ensureUserConfig = async (): Promise<void> => {
     await newUserConfig.$commit();
   }
 
+  commandObj.logToStderr(
+    `If you change your mind later, modify "countlyConsent" property in ${newUserConfig.$getPath()}`,
+  );
+
+  await handleDocsInConfigs(newUserConfig);
```

Comment:
What does `handleDocsInConfigs` do? Does it set config? Does it commit it to disk?

It doesn't return anything, yet it should have some effect, and then it is weird, because you do something implicit in `handleDocsInConfigs` and then something implicit in `setUserConfig`, and I guess their effects interdepends, and you can't call them in any order, right?

I'd suggest to
1. rename `handleDocsInConfigs` to something specific
2. make it return new value
3. use that value in `setUserConfig`